src/dataexplorer/dataexplorer-application/src/components/SchemaResponse/Schemes/Schemes.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { useState, useEffect } from 'react';
+import { Collapse, Row, Popover, Button } from 'antd';
+import './Schemes.style.scss';
+import {
+    IQuestionAndAnswer,
+    IQuestion,
+    IAnswer,
+} from 'types/IQuestionAndAnswer';
+import dayjs, { Dayjs } from 'dayjs';
+
+export interface SchemesProps {
+    questions: IQuestion[];
+    answers: IAnswer[];
+    questionnaireResource: fhir.ValueSet[];
+    date: Dayjs;
+    title: string;
+}
+
+const Schemes = (props: SchemesProps) => {
+    dayjs.locale('nb');
+    const [qAndA, setQAndA] = useState<IQuestionAndAnswer[]>([]);
+    useEffect(() => {
+        let hasAddedId = false;
+        // eslint-disable-next-line
+        props.questions.map((q) => {
+            // eslint-disable-next-line
+            props.answers.map((a) => {
```

Comment:
ESlint issue? What is the problem? Preferably this should be resolved instead of disabling ESLint.

Reply:
The map want's returns, but we don't want it to return anything. Could have empty returns, but then we could add code that did not have any purpose. @erlinvan and I had a discussion on this and concluded with adding eslint-disable

Reply:
What about this?

https://stackoverflow.com/questions/45014094/how-do-i-fix-expected-to-return-a-value-at-the-end-of-arrow-function-warning

Reply:
Yes, can use forEach instead 👍